core/domain/exp_domain.py

```diff
@@ -81,9 +81,18 @@
 CMD_RENAME_STATE = 'rename_state'
 # This takes an additional 'state_name' parameter.
 CMD_DELETE_STATE = 'delete_state'
+# DEPRECATED: This command is deprecated. Please do not use. The command remains
+# here to support old suggestions.
 # This takes additional 'state_name', 'content_id', 'language_code' and
```

Comment:
Optional: Perhaps wrap this around to the previous line, so that the DEPRECATED paragraph comes immediately before the cmd in question.

Reply:
Done.